### ISSUE TITLE
converting to c++ 

### ISSUE THREAD

**Issue by roblad** [mkorz/Integra#1]
Hi,

Is it possible to convert that code to c++ .h and .cpp files for compiling in linux ?

**Comment by mkorz** [comment]
I am certain it is possible, however, except for the brief encounter with C++ whilst doing my masters I do not have much experience with C++.
I could convert the project to .NET Core though, which would run on Linux if this would be of interest to you?
Alternatively, there is a sibling (more advanced) project written in Python, which runs on Linux just fine.

**Comment by roblad** [comment]
Yes /Tak
I want to monitor the ETHM-1 through RPI 2 Jeese Debian 8, if it will work after compilation it wil be ok, If you can prepare it I will appreciate.
But hhaw to run
Thanks

**Comment by etheaven** [comment]
@mkorz still planning to convert it to .NET Core ?

**Comment by mkorz** [comment]
Good question, I am still considering this, just not sure if there is enough interest and demand.
The Python version seemed to be much more popular and I plan to add more features once I sort out (integration) test environment, so I can play with more advanced commands without triggering alarm ;)

Are you interested specifically in .NET Core version?

**Comment by etheaven** [comment]
Yeah I kinda prefer Core. Well, the .NET version is very weirdly made(sorry) in my opinion and as I'm not proficient in python it gets harder a little. I just have the module without any other software and I want to controll it somehow so yeah, anything helps
